routines/example_with_inputs.openbb

```diff
@@ -1,5 +1,5 @@
 # This script requires you to use arguments. This can be done with the following:
-# exe routines/example_with_inputs.openbb -i TSLA,AAPL,MSFT
```

Comment:
Do we not need the `routine` folder location here?

Reply:
We do not, it automatically checks that. Buuuuut we will need to change the code when we move around the user folders to the `Documents` folder (WIP).